templates/run-benchmark.sh

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env bash
+# vim: ft=sh
+set -e
+
+export NIMBUS_ETH1_BENCHMARKS_REPO="/data/status-im/nimbus-eth1-benchmarks"
+export NIMBUS_ETH1_REPO="/data/status-im/nimbus-eth1"
+export NIMBUS_ETH1_TEMPLATE_DB="/data/template-db"
+export NIMBUS_ETH1_DB_DIR="${NIMBUS_ETH1_REPO}/data/shared_mainnet_0"
+export BENCHMARK_FILE_NAME='blocks-import-benchmark.csv'
+export HOST_BENCHMARKS_DIR='/data/host-benchmarks'
```

Comment:
Those are just hardcoded regardless of values of variables like `nimbus_eth1_service_path`?

Some of those should probably be derived from a root path for the service which should be templated in service definition right?

Reply:
Thanks for this suggestion.